CONTRIBUTING.md

```diff
@@ -165,7 +201,7 @@ npx lerna publish from-git # add `--dist-tag next` for pre-release versions
 rm $HOME/Downloads/packages-*
 ```
 
-### Updating the Java test server proto files
+## Updating the Java test server proto files
```

Comment:
In hindsight as pointed out by @cretz I wish I'd just copied the files over but it's fine for now.